android/modules/filesystem/src/java/ti/modules/titanium/filesystem/FilesystemModule.java

```diff
@@ -81,6 +81,10 @@ public boolean isExternalStoragePresent()
 	@Kroll.method
 	public FileProxy getFile(KrollInvocation invocation, Object[] parts)
 	{
+		//If directory doesn't exist, return
+		if (parts[0] == null) {
```

Comment:
Just out of curiosity: Maybe `Arrays.asList(parts).subList(0, parts.length).contains(null)` or a manual loop?